MarkdownPP/Modules/Include.py

```diff
@@ -90,6 +91,6 @@ def include(self, match, pwd=""):
             return data
 
         except (IOError, OSError) as exc:
-            print(exc)
+            logging.warning(exc)
 
```

Comment:
In these cases, you can just do `logging.exception("some message")` and it will automatically print the current exception.  This also means you don't need the `as exc` part when catching exceptions.